hw/ip/spi_host/doc/_index.md

```diff
@@ -341,6 +344,48 @@ This time delay is a half SCK cycle by default but can be extended to as long as
 
 These settings are all minimum bounds, and delays in the FSM implementation may create more margin in each of these timing constraints.
 
+### Idle Time Delays When Changing Configurations
+
+It is important that the configuration changes are applied while `csb` is high to avoid sending spurious `sck` events to either device.
+For example, if two devices have different requirements for `CPOL`, the clock polarity should not toggle except when `csb` is high (inactive) for all devices.
+
+Furthermore, `csb` should be remain high for the minimum idle time both before and after the configuration update.
+For example, consider a SPI_HOST attached to two devices each with different requirements for the clock divider, clock polarity, and idle time.
+Consider a configuration where total idle time (as determined by the {{< regref "CONFIGOPTS_0.CLKDIV_0" >}} and {{< regref "CONFIGOPTS_0.CSNIDLE_0" >}} multi-registers) works out to 9 idle clocks for the first device, and 4 clocks for second device.
+In this scenario then, when swapping from the first device to the second, the SPI_HOST IP will only swap the clock polarity once the first `csb` line, `csb[0]`, has been high for at least 9 clocks, and will continue to hold the second `csb` line, `csb[1]`, high for 4 additional clocks before starting the next transaction.
```

Comment:
Super-nit: Maybe "... swap the clock polarity **when** the first ...". When I read this quickly, I initially thought you were saying how many times the clock polarity was swapped.

Reply:
I actually massaged this sentence for a while, knowing that good old @rswarbrick would insist on clarity with conciseness.    ;)  Reading it myself I now feel like I've created a monster in the midst of my self doubt.  I'll take a stab at your phrasing.